aspnetcore/tutorials/index.md

```diff
@@ -5,19 +5,21 @@ description: A list of step-by-step guides for learning how to develop ASP.NET C
 keywords: ASP.NET Core,
 ms.author: tdykstra
 manager: wpickett
-ms.date: 10/14/2016
+ms.date: 7/14/2017
 ms.topic: get-started-article
-ms.assetid: a5313c5c-fdee-4ec6-9920-d94de5746e9b
 ms.technology: aspnet
 ms.prod: asp.net-core
 uid: tutorials/index
 ---
 # ASP.NET Core tutorials
 
+[Razor Pages](xref:mvc/razor-pages/index) is the recommended approach to create a Web UI with ASP.NET Core 2.0.
```

Comment:
This seems out of place here.  If you have to say it, how about as a parenthetical after the Razor pages tutorial entry which is already first in the web sites list?

Or put it under **Building web applications** and say there are two ways, MVC and RP, RP is the recommended way starting in AS.NET 2.0 or something like that.